packages/survey-creator-knockout/src/components/question.html

```diff
@@ -12,22 +13,28 @@
         data-bind="event: {pointerdown: (model, event)=>{startDragSurveyElement(event)}}"
       ></div>
     </div>
+    <!-- ko template: { name: 'svc-question-adoner-content', data: $data } --><!-- /ko -->
 
-    <!-- ko if: isEmptyElement -->
-    <div class="svc-panel__placeholder_frame">
-      <div
-        class="svc-panel__placeholder"
-        data-bind="text: placeholderText"
-      ></div>
-    </div>
-    <!-- /ko -->
-
-    <!-- ko component: { name: 'sv-template-renderer', params: { componentData: null, templateData: templateData } } -->
+    <!-- ko if: adornerComponent -->
+    <!-- ko component: { name: adornerComponent, params: { model: $data } } --><!-- /ko -->
     <!-- /ko -->
 
     <div class="svc-question__content-actions">
       <!-- ko component: { name: 'sv-action-bar', params: { model: actionContainer } } -->
       <!-- /ko -->
     </div>
   </div>
+  <!-- /ko -->
+  <!-- ko ifnot: question.isInteractiveDesignElement -->
+  <!-- ko template: { name: 'svc-question-adoner-content', data: $data } --><!-- /ko -->
+  <!-- /ko -->
 </div>
+<script type="text/html" id="svc-question-adoner-content">
+  <!-- ko component: { name: 'sv-template-renderer', params: { componentData: null, templateData: templateData } } -->
+  <!-- /ko -->
+  <!-- ko if: isEmptyElement -->
+  <div class="svc-panel__placeholder_frame">
+    <div class="svc-panel__placeholder" data-bind="text: placeholderText"></div>
+  </div>
+  <!-- /ko -->
+</script>
```

Comment:
My idea was to use component-wrapper for the common question wrapper functionality and use inner nodes for specific questions:
```html
<!-- ko component: { name: 'svc-question', params: { model: $data } } -->
<!-- specific question markup, e.g. for image -->
<div class="svc-image-question-controls">
   <input type="file" accept="image/*" class="svc-choose-file-input" />
   <sv-svg-icon
     class="svc-image-question-controls__button"
     data-bind="click: chooseFile"
     params="iconName: 'icon-file', size: 24"
   ></sv-svg-icon>
</div>
<!-- /ko -->
```
Of course our goal can be achieved via different approaches